libs/clients/national-registry/v3/src/lib/nationalRegistryV3.service.ts

```diff
@@ -0,0 +1,97 @@
+import { Injectable } from '@nestjs/common'
+import {
+  EinstaklingarApi,
+  EinstaklingurDTOAllt,
+  EinstaklingurDTOFaeding,
+  EinstaklingurDTOForsjaItem,
+  EinstaklingurDTOHeimili,
+  EinstaklingurDTOHju,
+  EinstaklingurDTOItarAuka,
+  EinstaklingurDTOLogforeldrar,
+  EinstaklingurDTOLoghTengsl,
+  EinstaklingurDTONafnAllt,
+  EinstaklingurDTORikisfang,
+  EinstaklingurDTOTru,
+  GerviEinstaklingarApi,
+} from '../../gen/fetch'
+import { isDefined } from '@island.is/shared/utils'
+
+@Injectable()
+export class NationalRegistryV3ClientService {
+  constructor(
+    private individualApi: EinstaklingarApi,
+    private fakeApi: GerviEinstaklingarApi,
+  ) {}
+
+  getAddress = (nationalId: string): Promise<EinstaklingurDTOHeimili | null> =>
+    this.individualApi.midlunV02EinstaklingarNationalIdHeimilisfangGet({
+      nationalId,
+    })
+
+  async getAllDataIndividual(
+    nationalId: string,
+  ): Promise<EinstaklingurDTOAllt | null> {
+    return this.individualApi.midlunV02EinstaklingarNationalIdGet({
+      nationalId,
+    })
+  }
```

Comment:
We should not use arrow function as class methods as they use more memory because each class instance has its own copy of the function instead of on the prototype.

More material:
https://www.typescriptlang.org/docs/handbook/2/classes.html#arrow-functions
 and
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#cannot_be_used_as_methods

Reply:
ah ok, will fix